app/hotels/src/HotelsContext.js

```diff
@@ -205,6 +207,4 @@ export const withHotelsContext = (select: (state: State) => Object) =>
 
 export type HotelsContextState = State;
 
-const HotelsContext = { Consumer, Provider };
-
-export default HotelsContext;
+export default { Consumer, Provider };
```

Comment:
In the other PR, you only exported Provider I believe. Not sure if it's important!

Reply:
Yes, I hope I will get there, but I don't want to refactor all components in one PR. There is around 30 files connecting to this context.